introduction/what-is.md

```diff
@@ -0,0 +1,45 @@
+
+# What is InnerSource?
+
+The basic idea of InnerSource is this: If a consuming team can't get what it needs by submitting a feature request then it may submit a pull request instead.
+The producing team partners to review and accept the pull request and the functionality it implements.
+In this context of a code submission, we call the consuming team the **Guest** team and the producing team the **Host** team.
+
+The terms **Guest** and **Host** suggest a situation analogous to receiving a visitor in the home.
+In that situation, most people want to be a good host.
+They ensure that things are kept neat and tidy in anticipation of their guests' arrival.
+Visitors are greeted at the door and invited in.
+They can use the features and utilities that are in the home's public areas.
+There may be a few house rules that guests are asked to follow.
+Similarly, most guests want to show respect for the home and their host.
+They're careful with the items in the house and follow the rules for the duration of their stay.
+They may hope for or expect a return invitation as long as they've been courteous and polite.
+These concepts around a home visit are a metaphor for the attitude and behaviors that teams should bring as one hosts another making a guest contribution to the codebase.
+
+Let's take a closer look at how the mechanics of the inner source process can work.
+To aid in this explanation, we'll name a few key individuals on the guest and host teams.
+First, the **Product Owner** determines what functionality the host team is willing to accept as a contribution.
+The **Contributor** is the individual that submits the code contribution.
+The **Trusted Committer** provides any timely support and mentorship that the contributor needs to successfully submit the pull request.
+
+With those definitions, here is the basic outline of an inner source contribution.
+
+* Guest team requests a feature from the host team.
+* Product owner ensures that user stories representing the feature request are created.
+These stories should describe the requested feature in terms agreeable to the guest team.
+They also list any details from the host team on how the feature should be delivered in order for the work to be accepted.
+Examples of such details include architecture constraints, coding conventions, dependency usages, data contracts, etc.
+* Supported by the trusted committer, the contributor submits the pull request to implement the requested feature.
+
+Note that these steps do not assume a specific system for the general organization of a team's time or priorities. Inner source assumes that teams already have existing such methods of organization and provides a framework of how to use them to work together where there is a guest team desiring to contribute code to a host.
+
+This option works well for the guest team because they get the functionality they need when they need it without taking on the long-term burden of maintenance of the solution.
```

Comment:
It only works if the _Trusted Committers_ do their job _and_ do it in a **timely fashion**, as you mentioned above. If they don't, the guest team would be back on square one. Maybe reference the trusted committer article here (instead of in the section _Additional Resources_, below.

Reply:
I agree that everyone doing their job is critical to making this process work.  Product Owners, too.  In general we'd like for all kinds of links to exist between these articles.  I'm not exactly sure how to do it.  I think that we'd want for the articles on O'Reilly to link to each other and the articles that we host on _innersourcecommons_ to link to each other.

I'm thinking the best thing  to do here is just remove the **Additional References** section and put a ticket in our _Kanban_ board to go and add in everything linking to each other after we're done producing it all.  What do you think?

Reply:
I'm thinking the best thing  to do here is just remove the **Additional References** section and put a ticket in our _Kanban_ board to go and add in everything linking to each other after we're done producing it all.  What do you think?

Reply:
good idea, @rrrutledge

Reply:
I'm very much for that idea too. 
Three things: 
- We're waiting for O'Reilly to come back to us with with how the like to have links annotated. I'd would hope that they like the Markdown annotation way ```[this](url)``` - if they don't have any preferences, I propose to suggest that to them.
- We still need to add all the hyperlinks we want then. Added a ticket for that since we probably would need to go through all the files once we know how to do it. See #41.
- The case of expectation management of collaboration through time zones that @gruetter mentioned is a case I  can confirm. It should be mentioned somewhere - probably both sides shortly of this effect on the TC and contributor segment - but I'm more for keeping it out of here.